src/core/reporters/TapReporter.js

```diff
@@ -247,9 +247,9 @@ export default class TapReporter {
 
     this.log(`1..${runEnd.testCounts.total}`);
     this.log(`# pass ${runEnd.testCounts.passed}`);
-    this.log(kleur.yellow(`# skip ${runEnd.testCounts.skipped}`));
-    this.log(kleur.cyan(`# todo ${runEnd.testCounts.todo}`));
-    this.log(kleur.red(`# fail ${runEnd.testCounts.failed}`));
+    this.log(`# ${kleur.yellow('skip ' + runEnd.testCounts.skipped)}`);
```

Comment:
Why (here & below) use string concatenation instead of template literals?

Reply:
I tend to default to concatenation still, applying template literals only when it makes the code more readable. 

Perhaps the difference is small enough that it doesn't matter, and that it is instead worthwhile to have only style in use. Personally, I haven't reached that point yet, especially when it's nested, I still favour concatenation:

```js
var str = 'foo bar ' + y.func( 'quux ' + z );

var str = `foo bar ${ y.func( `quux ${ z }` ) }`;
```

The concat example is immediately obvious to me as one string, followed by two more strings with a function applied. The template string example is notably more complex and ambigious until you read very carefully what starts and stops where, and what it embeds/calls, distinguishing between variables and string literals that look rather similar. Especially when producing output where special symbols are involved such as here in TAP, which of the braces and parenthesis are literal and which are JS template syntax? There's just a lot of clutter among them when it's nested and short.

I think the first form has less room for mistakes in writing, less ways to (mis)interpret it while reading, and quicker/easier to understand with confidence exactly what it does.

But, I'm definitely amplifying a small difference here for the purpose of answering your question "Why". I don't strongly oppose either style and would leave it as-is and unenforced based on what feels right.

Reply:
Changed it. Mainly because I realize it was that way already, and keeping it that way minimises the diff :)